src/tests/system/tests/test_ipa.py

```diff
@@ -0,0 +1,84 @@
+    """
+    :title: Add support to verify authentication
+     indicators in pam_sss_gss
+    :setup:
+        1. Add pam_sss_gss configuration to /etc/sssd/sssd.conf
+        2. Add pam_sss_gss.so to /etc/pam.d/sudo
+        3. Restart SSSD
+        4. Enable SSSD debug logs
+    :steps:
+        1. Switch to 'admin' user
+        2. obtain Kerberos ticket and check that it
+           was obtained using SPAKE pre-authentication.
+        3. Create sudo configuration that allows an admin to
+           run SUDO rules
+        4. Try 'sudo -l' as admin
+        5. As root, check content of sssd_pam.log
+       `6. Check if acquired service ticket has req. indicators: 0
+       `7. Add pam_sss_gss configuration to /etc/sssd/sssd.conf
+       `8. Check if acquired service ticket has req.
+           indicators: 2
+    :expectedresults:
+        1. Should succeed
+        2. Should succeed
+        3. Should succeed
+        4. Should succeed
+        5. Should succeed
+        6. Should succeed
+        7. Should succeed
+        8. Should succeed
+    :customerscenario: True
+    """
+    user = "admin"
+    test_password = "Secret123"
+
+    client.sssd.config["pam"] = {
+        "pam_gssapi_services": "sudo, sudo-i",
+        "pam_gssapi_indicators_map": "hardened, sudo:pkinit, sudo-i:otp",
+    }
+    client.sssd.start()
+    for path in ["/etc/pam.d/sudo", "/etc/pam.d/sudo-i"]:
+        client.fs.sed(path=path, command="2s/^/auth sufficient pam_sss_gss.so debug\\n/", args=["-i"])
+    client.host.ssh.run("sssctl config-check", raise_on_error=False)
+
+    client.host.ssh.run(f'su -l {user} -c "kinit admin@IPA.TEST"', input=test_password)
+    client.host.ssh.run(f'su -l {user} -c "klist"', raise_on_error=False)
+    client.host.ssh.run(f'su -l {user} -c "ipa sudocmd-add ALL2"', raise_on_error=False)
+    client.host.ssh.run(f'su -l {user} -c "ipa sudorule-add testrule2"', raise_on_error=False)
+    client.host.ssh.run(
+        f"su -l {user} -c \"ipa sudorule-add-allow-command testrule2 --sudocmds 'ALL2'\"", raise_on_error=False
+    )
+    client.host.ssh.run(f'su -l {user} -c "ipa sudorule-mod testrule2 --hostcat=all"', raise_on_error=False)
+    client.host.ssh.run(f'su -l {user} -c "ipa sudorule-add-user testrule2 --users admin"', raise_on_error=False)
+    client.host.ssh.run(f'su -l {user} -c "sudo -l"', raise_on_error=False)
+    time.sleep(3)
+    log1 = client.fs.read(client.sssd.logs.pam)
+    client.sssd.config["pam"] = {"pam_gssapi_services": "sudo, sudo-i", "pam_gssapi_indicators_map": "sudo-i:hardened"}
+    client.sssd.clear(db=True, memcache=True, logs=True)
+    client.sssd.restart()
+    client.host.ssh.run("sssctl debug-level 9")
+    client.host.ssh.run(f'su -l {user} -c "kinit admin@IPA.TEST"', input=test_password)
+    client.host.ssh.run(f'su -l {user} -c "sudo -l"', raise_on_error=False)
+    time.sleep(3)
+    log2 = client.fs.read(client.sssd.logs.pam)
+    assert "indicators: 0" in log1
```

Comment:
This could use some explanation what indicators: 0 and indicators: 2 actually mean.